src/station_gui.cpp

```diff
@@ -240,6 +240,8 @@ class CompanyStationsWindow : public Window
 	GUIStationList stations{filter.cargoes};
 	Scrollbar *vscroll;
 	uint rating_width;
+	std::array<uint16_t, NUM_CARGO> stations_per_cargo_type; ///< Number of stations with a rating for each cargo type.
+	uint16_t stations_per_cargo_type_empty; ///< Number of stations without a rating.
```

Comment:
Empty doesn't quite make sense, if I'm understanding this right.
```suggestion
	uint16_t stations_per_cargo_type_no_rating; ///< Number of stations without a rating.
```